apps/www/styles/globals.css

```
@@ -377,4 +377,119 @@
+    width: 0%;
+  }
+
+  100% {
+    width: 100%;
+  }
+}
+
+.animate-progress {
+  animation: progress 3s ease-in-out infinite;
+}
+@keyframes drawLine {
```

Stylelint (17.12.0)

[error] Expected keyframe name "drawLine" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)